source/features/action-pr-link.tsx

```diff
@@ -29,4 +53,8 @@ void features.add(import.meta.url, {
 
 https://github.com/refined-github/refined-github/actions
 
+https://github.com/refined-github/refined-github/pull/6794
+
+https://github.com/refined-github/refined-github/pull/6797/commits
```

Comment:
It doesn't appear on this page. These "test urls" must work.

This works:
```suggestion
https://github.com/refined-github/refined-github/pull/6794/commits
```